news/2020-02-10-community-choice-2019-results.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title: "Community Choice 2019: Results"
+date: 2020-02-10 08:00:00 +0000
+---
+
+You've had your say, and now the votes are in! Check out which beatmaps were the Community Favorites of 2019!
+
+![](/wiki/shared/news/2020-01-23-community-choice-2019-voting-open/header.jpg)
+
+Just to remind you, the top-voted map for each mode will become the Community Favorite of 2019 and earn their mapper (or set host) a shiny badge detailing how much everybody loves their work, plus 6 months of osu!supporter to tie it all off. The top ten finalists of each mode will also receive 3 months of osu!supporter to commemorate their near victory.
+
+Lots of maps to churn through, so let's get straight into the results without delay:
+
+### osu!
+
+[![](/wiki/shared/news/2020-02-10-community-choice-2019-results/ghost-rule.jpg)](https://osu.ppy.sh/beatmapsets/413117)
+
+**1421** voters woke up to **[Awaken](https://osu.ppy.sh/users/1380253)**'s set for **[DECO\*27 - Ghost Rule](https://osu.ppy.sh/beatmapsets/413117)**.
+
+This should come as little surprise to anyone, since the set had been sitting at both the top of the favorites list and the most played beatmaps ranking for a solid quarter of a year.
```

Comment:
think this should be the part of the previous paragraph, just to cut down some of this vertical space

same thing happens for catch